staging/src/k8s.io/cloud-provider/node/helpers/taints.go

```diff
@@ -139,3 +139,102 @@ func addOrUpdateTaint(node *v1.Node, taint *v1.Taint) (*v1.Node, bool, error) {
 	newNode.Spec.Taints = newTaints
 	return newNode, true, nil
 }
+
+// RemoveTaintOffNode is for cleaning up taints temporarily added to node,
+// won't fail if target taint doesn't exist or has been removed.
+// If passed a node it'll check if there's anything to be done, if taint is not present it won't issue
+// any API calls.
+func RemoveTaintOffNode(c clientset.Interface, nodeName string, node *v1.Node, taints ...*v1.Taint) error {
```

Comment:
Were there unit tests for this function? Can we move them here too?

Reply:
Added test